### PR TITLE
Add Async Sample Code

### PR DESCRIPTION
**Pull request by waprin**

### CODE REVIEW

File: appengine/urlfetch/async/rpc_test.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        return_value=mock.Mock(status_code=200,
+                               content="I'm Feeling Lucky"))
+    urlfetch_mock.create_rpc = mock.Mock(
+        return_value=mock.Mock(get_result=get_result_mock))
+    response = app.get('/')
+    assert response.status_int == 200
+    assert "I'm Feeling Lucky" in response.body
+
+
+def mock_callback():
```
Comment (jonparrott) [inline appengine/urlfetch/async/rpc_test.py]: why not just use a `mock.Mock()`?

**Comment by jonparrott** [comment]
LGTM with one nit.

File: appengine/urlfetch/async/rpc_test.py (hunk 1 above)
Comment (waprin) [inline appengine/urlfetch/async/rpc_test.py]: I......don't know.

Comment (jonparrott) [inline appengine/urlfetch/async/rpc_test.py]: ¯\\\_(ツ)\_/¯

**Pull request merged**